references/cuegen/testdata/valid/enum.cue

```diff
@@ -0,0 +1,14 @@
+package test
+
+Enum: {
+	a: "abc" | "def" | "ghi"
+	b: 1 | 2 | 3
+	c: true | false
+	d: 1.1 | 2.2 | 3.3
+	e: "abc" | "def" | *"ghi"
+	f: 1 | *2 | 3
+	g: true | *false
+	// if default value is first enum, '*' will not be added
+	h: 1.1 | 2.2 | 3.3
```

Comment:
If we have multiple enums(>=3) in one field, actually we need a default value, otherwise the cue will failed to generate openapi. If the users do not specify the default value, maybe we need to set the first enum as the default value.

Reply:
- Even if Generator marks the first value as the default, `cueformat.Simplify()` will remove `*`.

- It seems that enum without a default value is legal for openapi generation.

Test code:
```go
s := `
	#Def: {
		h: 1.1 | 2.2 | 3.3
		i: 1 | *2 | 3
	}
`
r := cuecontext.New()
bytes, err := openapi.Gen(r.CompileString(s), nil)
if err != nil {
	panic(err)
}
fmt.Println(string(bytes))
```

Result:
```json
{
  "openapi": "3.0.0",
  "info": {
    "title": "Generated by cue.",
    "version": "no version"
  },
  "paths": {},
  "components": {
    "schemas": {
      "Def": {
        "type": "object",
        "required": [
          "h",
          "i"
        ],
        "properties": {
          "h": {
            "type": "number",
            "enum": [
              1.1,
              2.2,
              3.3
            ]
          },
          "i": {
            "type": "integer",
            "enum": [
              2,
              1,
              3
            ],
            "default": 2
          }
        }
      }
    }
  }
}

```
<img width="343" alt="image" src="https://user-images.githubusercontent.com/61452000/222380415-c7f9528d-5bb3-4852-a93a-a7c30520792d.png">

Reply:
Hi, any suggestion or review?